sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ExpressionParserSuite.scala

```diff
@@ -263,21 +263,60 @@ class ExpressionParserSuite extends PlanTest {
       "sum(product + 1) over (partition by ((product / 2) + 1) order by 2)",
       WindowExpression('sum.function('product + 1),
         WindowSpecDefinition(Seq('product / 2 + 1), Seq(Literal(2).asc), UnspecifiedFrame)))
+  }
+
+  test("range/rows window function expressions") {
```

Comment:
How about `window function expressions on various window frames`?

Reply:
Just back from holiday. What do you mean exactly here? Range/rows between test with most of the combinations. What is missing?

Reply:
@jiangxb1987 gentle ping

Reply:
sorry for the late response. Just update the test case name.